moveit_kinematics/kinematics_base_test/CMakeLists.txt

```diff
@@ -0,0 +1,41 @@
+cmake_minimum_required(VERSION 2.8.3)
+project(kinematics_base_test)
+
+add_compile_options(-std=c++11)
+
+find_package(catkin REQUIRED COMPONENTS
+  moveit_core
+  pluginlib
+  rostest
+  rosunit
+  moveit_ros_planning
+  moveit_kinematics
+)
+
+find_package(Boost REQUIRED COMPONENTS system)
+
+###################################
+## catkin specific configuration ##
+###################################
+catkin_package(
+)
+
+###########
+## Build ##
+###########
+
+include_directories(
+  ${catkin_INCLUDE_DIRS}
+)
+
+#############
+## Testing ##
+#############
+
+add_rostest_gtest(${PROJECT_NAME}_utest launch/test_kinematics_plugin.launch src/test_kinematics_plugin.cpp)
```

Comment:
you need to wrap the testing stuff with a if statement like here:

https://github.com/ros-planning/moveit/blob/melodic-devel/moveit_core/robot_state/CMakeLists.txt#L20